src/mumble/VolumeSliderWidgetAction.h

```diff
@@ -0,0 +1,33 @@
+// Copyright 2005-2020 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#ifndef MUMBLE_MUMBLE_VOLUMEWIDGETACTION_H_
+#define MUMBLE_MUMBLE_VOLUMEWIDGETACTION_H_
+
+
+#include <QMap>
+
+#include "ClientUser.h"
+
+#include <QtWidgets/QSlider>
```

Comment:
Please move the include to the cpp file and use a forward declaration in the header instead.